android/src/main/kotlin/com/hypertrack/sdk/flutter/FlutterSerialization.kt

```diff
@@ -0,0 +1,52 @@
+package com.hypertrack.sdk.flutter
+
+import com.hypertrack.sdk.flutter.common.*
```

Comment:
I'm not a fan of wildcard imports, it can be easily messed up with classes that are named similarly

Reply:
fixed